HW2/P2/P2.py

```diff
@@ -18,9 +19,12 @@
     orig_counts = np.arange(1000, dtype=np.int32)
     src = np.random.randint(1000, size=1000000).astype(np.int32)
     dest = np.random.randint(1000, size=1000000).astype(np.int32)
-
     total = orig_counts.sum()
 
+    correlated = []
+    uncorrelated = []
+    N_list = [5, 10, 20, 30, 50, 75, 100]    
```

Comment:
I did the same thing. But one of my reviewer asked me to try greater values. like 1000, not sure if it's necessary